### PR TITLE
Use `autoload-dev` setting on "composer.json" file

### PR DESCRIPTION
**Pull request by henriquemoody**
This also removes "tests/bootstrap.php" file and defines `date.timezone` in "phpunit.xml.dist".

See the discussion [starting on #356](https://github.com/Seldaek/monolog/pull/356#issuecomment-40899938).

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<phpunit bootstrap="tests/bootstrap.php" colors="true">
```
Comment (stof) [inline phpunit.xml.dist]: the bootstrap  file should be set to ``vendor/autoload.php``

Comment (henriquemoody) [inline phpunit.xml.dist]: No problem, fixed.

Thanks

**Comment by stof** [comment]
:+1:

**Comment by Seldaek** [comment]
Thanks

**Pull request merged**